src/dashboard/elements/ncg-dashboard.html

```diff
@@ -200,6 +200,11 @@
 				display: none !important;
 			}
 
+			paper-tab[hidden] {
```

Comment:
Good catch! This can be modified to be just a general `[hidden]` style (i.e., remove the `paper-tab` part) to ensure that `[hidden]` styles are property applied to all elements in `ncg-dashboard`.